cclient/response/addresses_spent_from.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 IOTA Foundation
+
+#ifndef CCLIENT_RESPONSE_ADDRESSES_SPENT_FROM_H
+#define CCLIENT_RESPONSE_ADDRESSES_SPENT_FROM_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "types/types.h"
+
+typedef struct {
+  int_array states;
+} addresses_spent_from_res_t;
```

Comment:
the actual api call is called `wereAddressesSpentFrom`

```
                case "wereAddressesSpentFrom": {
                    final List<String> addresses = getParameterAsList(request,"addresses", HASH_SIZE);
                    return wereAddressesSpentFromStatement(addresses);
                }
```

Could you adjust the naming here & of the files?

Reply:
done